### PR TITLE
Adds magic types from fantasy novels and games

### PR DESCRIPTION
**Pull request by Alberdi**

### CODE REVIEW

File: data/games/fantasy_magics.json [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+			}]
+		},
+		{
+			"name": "matemagia",
+			"gender": "f",
+			"practitioners": [{
+				"name": "matemágica",
+				"gender": "f"
+			}, {
+				"name": "maetmágico",
```
Comment (xurxodiz) [inline data/games/fantasy_magics.json]: *matemágico

**Review by xurxodiz — CHANGES_REQUESTED** [review]

File: data/games/fantasy_magics.json (hunk 1 above)
Comment (xurxodiz) [inline data/games/fantasy_magics.json]:
```suggestion
				"name": "matemágico",
```

**Review by xurxodiz — CHANGES_REQUESTED** [review]

File: data/games/fantasy_magics.json [hunk 2]
```diff
@@ -0,0 +1,341 @@
+{
+	"description": "Modalidades fantásticas de magia y sus practicantes",
+	"magics": [{
+			"name": "abjuración",
+			"gender": "l",
```
Comment (xurxodiz) [inline data/games/fantasy_magics.json]:
```suggestion
			"gender": "f",
```

File: data/games/fantasy_magics.json [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+				"name": "chamana",
+				"gender": "f"
+			}, {
+				"name": "chamán",
+				"gender": "m"
+			}]
+		},
+		{
+			"name": "conjuración",
+			"gender": "l",
```
Comment (xurxodiz) [inline data/games/fantasy_magics.json]:
```suggestion
			"gender": "f",
```

**Review by xurxodiz — CHANGES_REQUESTED** [review]

File: data/games/fantasy_magics.json (hunk 2 above)
Comment (Alberdi) [reply data/games/fantasy_magics.json]: 😞

**Review by xurxodiz — APPROVED** [review]

**Pull request merged**